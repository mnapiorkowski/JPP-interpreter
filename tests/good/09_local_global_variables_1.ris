global int x = 42;

void glob_var():
    x = x + 1;
    println x; // 43
    return;

void loc_var():
    let int x = 10;
    glob_var();
    println x; // 10
    return;

void main():
    println x; // 42
    loc_var();
    println x; // 43
    return;