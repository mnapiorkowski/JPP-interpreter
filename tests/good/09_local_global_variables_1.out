42
11
11
42
